cluster_management/src/main/java/com/pinterest/rocksplicator/spectator/ConfigGeneratorClusterSpectatorFactory.java

```diff
@@ -0,0 +1,22 @@
+package com.pinterest.rocksplicator.spectator;
+
+public class ConfigGeneratorClusterSpectatorFactory implements ClusterSpectatorFactory {
+
+  private final String uriPattern;
+
+  public ConfigGeneratorClusterSpectatorFactory(String uriPattern) {
+    this.uriPattern = uriPattern;
+  }
+
+  private String getConfigPostUri(String uriPattern, String clusterName) {
+    return uriPattern.replace("[PARTICIPANT_CLUSTERR]", clusterName);
```

Comment:
-> Seems spelling mistake `s/CLUSTERR/CLUSTER/g`

Note: Here we replace the '[PARTICIPANT_CLUSTER]' with the actual helix participant cluster that needs to be managed by this leader spectator.